### PR TITLE
-adding tests for makedir and delete dir and also remove unused names…

### PR DESCRIPTION
**Pull request by github-ccampian**
-adding tests for makedir and delete dir and also remove unused names…

### CODE REVIEW

File: src/test/java/org/mule/extension/hdfs/automation/functional/DeleteDirectoryTestCases.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      flowRunner(Util.FlowNames.DELETE_DIR_FLOW).withVariable("path", PARENT_DIRECTORY + NEW_DIRECTORY)
+      .run();
+      verifyDeletetionOfDirectory();
+  }
+
+    private void verifyDeletetionOfDirectory() throws Exception {
+        fileNotFoundExpected.expect(Exception.class);
+        List<FileStatus> listStatus = (List<FileStatus>) TestDataBuilder
+                .getValue(flowRunner(Util.FlowNames.LIST_STATUS_FLOW).withVariable("path", PARENT_DIRECTORY + NEW_DIRECTORY)
+                        .run());
```
Comment (DBCSV) [inline src/test/java/org/mule/extension/hdfs/automation/functional/DeleteDirectoryTestCases.java]: why do we expect error here?

File: src/test/java/org/mule/extension/hdfs/automation/functional/MakeDirectoriesTestCases.java [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        return Util.MAKE_DIR_FLOW_PATH;
+    }
+
+    @Test
+    public void testMakeDirectories() throws Exception {
+        flowRunner(Util.FlowNames.MAKE_DIR_FLOW).withVariable("path", PARENT_DIRECTORY + NEW_DIRECTORY)
+                .withVariable("permission", "700")
+                .run();
+
+        List<FileStatus> parentDirectoryStatuses = (List<FileStatus>) TestDataBuilder
```
Comment (DBCSV) [inline src/test/java/org/mule/extension/hdfs/automation/functional/MakeDirectoriesTestCases.java]: move the `assertion` phase to a different method as in `DeleteDirectoryTestCases` for consistency

File: src/test/java/org/mule/extension/hdfs/automation/functional/ReadOperationTestCases.java [hunk 3]
```diff
@@ -59,7 +59,7 @@ public void testReadFlow() throws Exception {
     public void testReadOpFlowUnexistingFile() throws Exception {
 
         expectedException.expect(MessagingException.class);
-        expectedException.expectMessage(StringContains.containsString("Something went wrong with input data"));
+        expectedException.expectMessage(StringContains.containsString("Invalid request data"));
```
Comment (DBCSV) [inline src/test/java/org/mule/extension/hdfs/automation/functional/ReadOperationTestCases.java]: Make this a constant available in `test` module so we can reuse it. Also I think we can verify the full exception message.

File: src/test/java/org/mule/extension/hdfs/automation/functional/DeleteDirectoryTestCases.java (hunk 1 above)
Comment (github-ccampian) [inline src/test/java/org/mule/extension/hdfs/automation/functional/DeleteDirectoryTestCases.java]: Liststatus operation throws a FileNotFoundException - when the path does not exist; and we just removed the path...

File: src/test/java/org/mule/extension/hdfs/automation/functional/MakeDirectoriesTestCases.java (hunk 2 above)
Comment (github-ccampian) [inline src/test/java/org/mule/extension/hdfs/automation/functional/MakeDirectoriesTestCases.java]: Done

File: src/test/java/org/mule/extension/hdfs/automation/functional/ReadOperationTestCases.java (hunk 3 above)
Comment (github-ccampian) [inline src/test/java/org/mule/extension/hdfs/automation/functional/ReadOperationTestCases.java]: Done

**Pull request merged**